bundles/org.openhab.binding.tplinkrouter/src/main/resources/OH-INF/thing/channel-group-types.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tplinkrouter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-group-type id="wifiGroupType">
+		<label>Wifi</label>
+		<description>Wifi channels</description>
+		<channels>
+			<channel id="status" typeId="switch-type">
+				<label>Wifi Status</label>
+				<description>Wifi status</description>
+			</channel>
+			<channel id="ssid" typeId="string-type">
+				<label>Wifi SSID</label>
+				<description>Wifi SSID</description>
+			</channel>
+			<channel id="bandwidth" typeId="bandwidth-type">
```

Comment:
What is generally done by everyone is to define a channel type with the same name as the channel itself.
So I would rather recommend that you name your channel types "bandwith", "authentication", ...